commit 1aa71c4ca347444622edc5cf0ebea39ba69718f0
Author: RahifM <rahifmanjatha372@gmail.com>
Date:   Fri Aug 19 19:48:29 2022 +0000

    Revert "libbfqio: Remove vendor variant"
    
    This reverts commit 506c0b73427a984f003da7674e06cc4109bd987f.

diff --git a/libbfqio/Android.bp b/libbfqio/Android.bp
index d9a7edbe..dd816293 100644
--- a/libbfqio/Android.bp
+++ b/libbfqio/Android.bp
@@ -1,5 +1,5 @@
 //
-// Copyright (C) 2017-2019 The LineageOS Project
+// Copyright (C) 2017 The LineageOS Project
 //
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
@@ -20,9 +20,8 @@ cc_library_headers {
     export_include_dirs: ["include"],
 }
 
-cc_library {
-    name: "libbfqio",
-    vendor_available: true,
+cc_defaults {
+    name: "libbfqio_defaults",
     srcs: [
         "bfqio.c",
     ],
@@ -42,3 +41,15 @@ cc_library {
         "-Wextra",
     ],
 }
+
+cc_library {
+    name: "libbfqio",
+    vendor_available: true,
+    defaults: ["libbfqio_defaults"],
+}
+
+cc_library {
+    name: "libbfqio_vendor",
+    vendor: true,
+    defaults: ["libbfqio_defaults"],
+}
